### DISCUSSION TITLE
在vercel下部署遇到504错误，在本地运行没有问题

### DISCUSSION THREAD

**Discussion by lovebai** [spencerwooo/onedrive-vercel-index#261] [Q&A] [answered]
本地可以正常运行，部署到vercel页面报504错误
![屏幕截图 2022-01-11 091224](https://user-images.githubusercontent.com/42148495/148863976-f96877b0-1e02-4671-9f2e-63b67093e50e.png)

**Comment by rinrinx2** [comment, marked as answer]
https://youtu.be/47cVvI2Kw1M

**Reply by lovebai** [reply]
Thanks, I put the link of redis wrong
